### PR TITLE
Add dynamic main title

### PR DESCRIPTION
**Pull request by sgammon**
- Add render function to decide what the widget's main title will say
- Add an appropriate main title string for the CFR tab

### CODE REVIEW

File: costtree-slider.html [hunk 1]
```diff
@@ -523,7 +523,7 @@
       <div class="ct-logo-container">
         <costtree-logo mode="vector" cobrand="ecivis"></costtree-logo>
       </div><!-- end div.ct-logo-container -->
-      <div class="slider-title slider-title-general">General Rules for Allowable Indirect Costs</div>
+      <div class="slider-title slider-title-general">[[_renderMainTitle(mode)]]</div>
```
Comment (sgammon) [inline costtree-slider.html]: render function invocation

File: costtree-slider.html [hunk 2]
```diff
@@ -960,6 +960,16 @@
...
 
       /**
+       * Utility function, used from local DOM to render the main widget title,
+       * depending on the current widget `mode`.
+       */
+      _renderMainTitle: function(mode) {
+        if (mode == "cfr")
+          return "Uniform Guidance Quick Reference";
+        return "General Rules for Allowable Indirect Costs";
+      },
```
Comment (sgammon) [inline costtree-slider.html]: new render function

**Comment by sgammon** [comment]
General tab remains unchanged:
screen shot 2016-07-07 at 11 19 07

CFR tab after this change:
screen shot 2016-07-07 at 11 19 15

**Comment by sgammon** [comment]
Updated component demo here: https://cost-tree.firebaseapp.com/elements/costtree-slider?view=demo:demo/index.html&active=costtree-slider

**Comment by janslaught** [comment]
LOOKS GREAT!

**Pull request merged**